oeps/processes/oep-0057-core-repositories.rst

```diff
@@ -0,0 +1,166 @@
+
+      * it has assigned `maintainers`_ who strive to:
+
+        * review community code contributions,
+        * participate in community discussions related to the repository, and
+        * keep the repository compliant with the accepted Open edX Proposals (OEPs).
+
+* We declare that all access to write, configure, or otherwise administer Core repositories shall be granted through the `Core Contributor Program`_ and managed by its Program Administrators.
+
+* We declare that the addition or removal of repositories from the Core should by reviewed by existing Core Contributors.
+
+  * TODO: Define a review process.
+
+
+Reasoning
+---------
+
+Many other repositories exist as part of the Open edX community. Some of these are forks of Core repositories, and others are upstream repositories in their own right. These various non-Core repositories are vital to different Open edX community members. Furthermore, the existence of non-Core repositories in general is important to the health of the Open edX project, which thrives on collaboration, experimentation, and extension.
+
+However, from these many repositories, we wish to enumerate a smaller set which:
+
+* is a good target for enhancement, documentation, and technical investment;
+
+* defines a logical boundary within which it is fair to enforce Open edX Proposals;
+
+* defines an administrative boundary within which the Core Contributor Program applies;
+
+* makes the Open edX project more comprehensible by collecting its most critical components;
+
+* (TODO: should I add something about tCRIL stewardship/ownership?); and
+
+* encompasses the first-party aspect of the Open edX release supply chain, identifying repositories that need to be:
+
+  * monitoried for discloures under the `Open edX security policy`_,
+  * regularly updated with security patches,
+  * covered by automated unit tests and static analysis, and
+  * ready to deploy to production at any time.
+
+
+Considerations
+--------------
+
+
+* Resolving legacy processes and access policies
+
+  * The process for reviewing addition/removal of repos via the Core Contributor program isn't ready to be spun up yet or even well-defined. In the meantime, tCRIL has been making these judgement calls.
+  * 2U OCM and some of tCRIL retain historical access to many Core repositories outside of the Core Contributor program process. There is a plan to bring those folks under the Core Contributor umbrella, but it will take time.
+  * tCRIL is currently using a few public repositories in the openedx organization for day-to-day work, such as ``openedx/tcril-engineering``. Upon acceptance of this OEP, they would need to make a new GitHub organization for those repositories.
+
+* Cleaning up the Core
+
+  * Many repositories in the openedx GitHub organization do not currently meet the Core criteria. We will need to review the organization contents, and for several repos, transfer them out or invest in them.
```

Comment:
Nit: here - and any other place you describe current state - could you possibly say something like `do not (at time of initial OEP authoring)` or `do not (as of early 2022)`? I find OEPs are often not revisited and language like this change, potentially leading people reading this in the future to think this sentence describes the "current state" at that moment in time.

Reply:
Yes, this entire section will be rewritten before the proper review period.